examples/contracts/src/lib.rs

```diff
@@ -248,7 +249,6 @@ mod tests {
             .await?;
 
         // ANCHOR_END: tx_parameters_default
-        // ANCHOR: tx_parameters
```

Comment:
I'm out of the loop -- why are we removing this anchor here?

Reply:
It's duplicated. If you look at the "Master" branch, you will see that ANCHOR: tx_parameters and ANCHOR_END: tx_parameters are repeated twice on lines 231 - 241 and 251 - 261. I didn't know which one to keep, so I deleted one so that someone could notice and decide which one to keep. :thinking:

Reply:
Nice, makes sense!